examples/optimize_triplet/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _examples-triplet:
+
+Optimized Triplet
+===================
+
+Optimization of focusing parameters for a quadrupole triplet.
+A 1 GeV electron beam is strongly focused from lattice initial parameters:
```

Comment:
The script initializes a 2 GeV electron beam - should it initialize a 1 GeV beam or does the value here need to be updated?